### PR TITLE
add prose linters

### PR DESCRIPTION
**Pull request by Konfekt**
In a similar vein to the syntax linters, these TeX prose linters might be of use for little effort.

### CODE REVIEW

File: compiler/style-check.vim [hunk 1]
```diff
@@ -0,0 +1,19 @@
+if exists("current_compiler") | finish | endif
+let current_compiler = "style-check"
+
+if exists(":CompilerSet") != 2		" older Vim always used :setlocal
```
Comment (lervag) [inline compiler/style-check.vim]: I prefer to put comments above instead of inline.

File: compiler/textidote.vim [hunk 2]
```diff
@@ -0,0 +1,28 @@
+if exists("current_compiler") | finish | endif
+let current_compiler = "textidote"
+
+if exists(":CompilerSet") != 2		" older Vim always used :setlocal
```
Comment (lervag) [inline compiler/textidote.vim]: I prefer to put comments above instead of inline.

File: compiler/textidote.vim [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  command -nargs=* CompilerSet setlocal <args>
+endif
+
+let s:cpo_save = &cpo
+set cpo&vim
+
+if exists('g:textidote_jar') && filereadable(fnamemodify(g:textidote_jar, ':p'))
```
Comment (lervag) [inline compiler/textidote.vim]: Is there a reason you do not use the `g:vimtex_...` naming scheme here?

Comment (Konfekt) [reply compiler/textidote.vim]: No, just an oversight on transforming this from a general compiler into on that becomes part of the vimtex plug-in.

**Comment by lervag** [comment]
Thanks for this!

**Pull request closed without merging**